crates/base_db/src/input.rs

```diff
@@ -205,6 +223,7 @@ pub struct CrateData {
     pub env: Env,
     pub dependencies: Vec<Dependency>,
     pub proc_macro: Vec<ProcMacro>,
+    pub origin: CrateOrigin,
```

Comment:
Mmh, we are actually duplicating a bit of data here now, `CrateData` already knows about a crate's version in the `version` field, but now we also track it in the origin here, same thing for `display_name` I think?

Reply:
Agreed. If I recall correctly, version wasn't there in my fork from two weeks ago. I'll fix it.